bot/engine/src/main/kotlin/admin/dialog/RatingReportQueryResult.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2017/2021 e-voyageurs technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.tock.bot.admin.dialog
+
+
+/**
+ *
+ */
+data class RatingReportQueryResult(
+    val ratingBot: Int?,
+    val nbUsersRated: Int?,
+    var ratingDetails: List<DialogRating> = emptyList(),
```

Comment:
use val instead of var - and use copy() when changing a value

Reply:
resolved